website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -35,11 +35,14 @@ than it would be in a traditional monolithic application.
 Since our html server is so simple,
 we'll build it using [ExpressJS](http://expressjs.com).
 Exosphere provides a template for building ExpressJS html servers.
-Let's use it:
+So, before we add the service,
+we will add the HTML server service template and then we will add the new service.
 
 <a class="runMarkdown_consoleWithInputFromTable">
+
```

Comment:
Why the empty line here?

Reply:
The code block doesn't render correctly via Markdown preview without the line break between the <a> and the opening of the code block (```).

Reply:
Okay, makes sense. Please let me know when this is ready to be looked at again.